controllers/common.go

```diff
@@ -127,12 +126,12 @@ func isAivenServerError(err error) bool {
 }
 
 // NewAivenClient returns Aiven client (aiven/aiven-go-client/v2)
-func NewAivenClient(token string) (*aiven.Client, error) {
+func NewAivenClient(token, version string) (*aiven.Client, error) {
 	return aiven.NewTokenClient(token, "k8s-operator/"+version)
 }
 
 // NewAivenGeneratedClient returns Aiven generated client client (aiven/go-client-codegen)
-func NewAivenGeneratedClient(token string) (avngen.Client, error) {
+func NewAivenGeneratedClient(token, version string) (avngen.Client, error) {
```

Comment:
Do we want to include the kube version or the operator version here or both? It would be good to have both versions in the user agent. Something like `k8s-operator/v0.17.0-k8s-1.26.0`. What do you think @byashimov?

Reply:
We need them both. Like in TF.